src/components/BookList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from './Book';
+
+const mapStateToProp = state => ({ books: state.bookReducer });
+
+const BookList = ({ books }) => {
+  const list = [];
+  books.forEach(book => {
+    list.push(
+      <Book key={book.ID} book={book} />,
+    );
```

Comment:
- [ ] Kindly move BookList to src/containers folder. It qualifies as a container since it isn't a presentational component.

Reply:
Done.